src/core/config.ts

```diff
@@ -6,5 +6,10 @@ export module Configs {
    * Specifies if Plottable should show warnings.
    */
   export var SHOW_WARNINGS = true;
+
+  /**
+   * Specifies if Plottable should add title elements.
```

Comment:
Probably expand this to "`Specifies if Plottable should add <title> elements to text.`" for clarity.